### PR TITLE
Fix #15514: Two 'quit to menu' menu items available in track designer

### LINKED ISSUE #15514: Track Designer Incorrect Quit
### Operating System

Windows 10, 64-bit

### OpenRCT2 build

OpenRCT2, v0.3.4.1 (b7bda6e on develop) provided by GitHub

### Describe the issue

In the Track Designer, the options dropdown menu incorrectly displays the quit option as "Quit Track Designs Manager" but functions correctly to exit to main menu. The dropdown also offers "Quit Track Designer" and while that is labeled correctly, it is not functional when selected.

image

### Area(s) with issue?

_No response_

### Steps to reproduce

1. Select Track Designer in the main menu
2. Go to options to quit to main menu
3. Quit option is mislabeled and correctly labeled option is nonfunctional

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by Broxzier**

### CODE REVIEW

File: src/openrct2-ui/windows/TopToolbar.cpp [hunk 1]
```diff
@@ -441,11 +441,11 @@ static void window_top_toolbar_mousedown(rct_window* w, rct_widgetindex widgetIn
...
                 gDropdownItemsFormat[numItems++] = STR_FILE_BUG_ON_GITHUB;
                 gDropdownItemsFormat[numItems++] = STR_EMPTY;
-                gDropdownItemsFormat[numItems++] = STR_QUIT_TRACK_DESIGNS_MANAGER;
-                gDropdownItemsFormat[numItems++] = STR_EXIT_OPENRCT2;
-
                 if (gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER)
                     gDropdownItemsFormat[numItems++] = STR_QUIT_ROLLERCOASTER_DESIGNER;
+                else
+                    gDropdownItemsFormat[numItems++] = STR_QUIT_TRACK_DESIGNS_MANAGER;
+                gDropdownItemsFormat[numItems++] = STR_EXIT_OPENRCT2;
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/TopToolbar.cpp:448]:
```suggestion

                gDropdownItemsFormat[numItems++] = STR_EXIT_OPENRCT2;
```

File: distribution/changelog.txt [hunk 2]
```diff
@@ -30,6 +30,7 @@
 - Fix: [#15476] Crash when placing/clearing small scenery.
 - Fix: [#15487] Map animations do not work correctly when loading an exported SV6 file in vanilla RCT2.
 - Fix: [#15496] Crash in paint_swinging_inverter_ship_structure().
+- Fix: [#15514] Two different 'quit to menu' menu items are available in track designer and track design manager.
```
Comment (Gymnasiast) [inline distribution/changelog.txt:33]:
```suggestion
- Fix: [#15514] Two different “quit to menu” menu items are available in track designer and track design manager.
```

**Comment by Gymnasiast** [comment]
Yes, I am nitpicky :P

**Comment by Broxzier** [comment]
Nitpicky is good!

**Pull request merged**